packages/playwright-core/types/har.d.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// see http://www.softwareishard.com/blog/har-12-spec/
```

Comment:
I'm scared about all these types in the language ports. Do we do it like storageState there to accept just a string?

Reply:
We'll see, these types haven't changed for a while so I believe we can expose them in the languages, we can stay on the safe side and just accept an opaque JSON object too but it'd be less convenient.